File: examples/modulith-clinical-tool-jpa/src/test/java/io/zenwave360/example/clinicaltool/modules/clinical/infrastructure/jpa/inmemory/HospitalRepositoryInMemory.java
package io.zenwave360.example.clinicaltool.modules.clinical.infrastructure.jpa.inmemory;

import io.zenwave360.example.clinicaltool.modules.clinical.core.domain.*;
import io.zenwave360.example.clinicaltool.modules.clinical.core.outbound.jpa.HospitalRepository;
import java.math.*;
import java.time.*;
import java.util.*;

public class HospitalRepositoryInMemory extends InMemoryJpaRepository<Hospital> implements HospitalRepository {}

File: examples/modulith-klinical-tool-jpa/src/test/kotlin/io/zenwave360/example/clinicaltool/modules/clinical/infrastructure/jpa/inmemory/HospitalRepositoryInMemory.kt
package io.zenwave360.example.clinicaltool.modules.clinical.infrastructure.jpa.inmemory

import io.zenwave360.example.clinicaltool.modules.clinical.core.domain.*
import io.zenwave360.example.clinicaltool.modules.clinical.core.outbound.jpa.HospitalRepository
import java.math.*
import java.time.*
import java.util.*

class HospitalRepositoryInMemory : InMemoryJpaRepository<Hospital, Long>(), HospitalRepository {}
